scripts/lib.py

```diff
@@ -96,7 +96,7 @@ def get_seed(seed):
   return random.getrandbits(32)
 
 
-def run_cmd(cmd, timeout_s = 999, exit_on_error = 1, check_return_code = True):
+def run_cmd(cmd, debug_cmd = None, timeout_s = 999, exit_on_error = 1, check_return_code = True):
```

Comment:
Can you move debug_cmd to the last argument to make it backward compatible?

Reply:
BTW, do you plan to add it to run_parallel_cmd?

Reply:
Yes, it does not finish yet. I will add it to run_parallel_cmd. And then I will change cov.py too. I just make a simple change to see how it works.

Reply:
For backward compatible, I can move debug_cmd to the last argument. But I am not sure about what do you mean `backward compatible`?

Reply:
I am worried some user might have some code like run_cmd(cmd, 1000), adding debug_cmd in the middle will break it.

Reply:
Okay I see